R/promptAndRun.R

```diff
@@ -70,10 +61,16 @@ promptAndRun <- function(mydir = ".", user = NULL, daysback = 3) {
 #    }
 #    message("Found these runs (only first 50 shown)")
     print(myruns[1:min(1500, length(myruns))])
```

Comment:
instead of the long string, I wrote individual columns that can be easily adjusted for their length by adding and deleting spaces. The length information is passed to `printOutput`, so that the two are always consistent